landoui/templates/signout.html

```diff
@@ -1,6 +1,8 @@
 {% extends "partials/layout.html" %}
+{% block page_title %}Signed Out - Lando@Mozilla{% endblock %}
```

Comment:
Let's axe the `@` and do something more "professional", like "Lando - Mozilla" or "Mozilla Lando"